src/mixin/entity-mixin-lit.js

```diff
@@ -85,11 +52,27 @@ export const EntityMixinLit = superclass => class extends superclass {
 		}
 	}
 
-	__onHrefChange(href, token) {
-		dispose(this._entity); // Make sure the entity is cleaned up before setting a new one.
+	_getEntity() {
+		dispose(this.entity);
 		if (typeof this._entityType === 'function') {
```

Comment:
This is an alternative approach to using `runAsync` that still gives us `pending-state` support.
These events are used by the `PendingContainer` mixin to track how many children are pending.
This could be used to render skeleton components in the main container until all the children are ready.